src/data/roadmap/releases.tsx

```diff
@@ -0,0 +1,194 @@
+import { StaticImageData } from "next/image"
+
+import CommunityHeroImage from "@/public/images/heroes/community-hero.png"
+import DevelopersHubHeroImage from "@/public/images/heroes/developers-hub-hero.jpg"
+import GuidesHubHeroImage from "@/public/images/heroes/guides-hub-hero.jpg"
+import Layer2HubHeroImage from "@/public/images/heroes/layer-2-hub-hero.jpg"
+import QuizzesHubHeroImage from "@/public/images/heroes/quizzes-hub-hero.png"
+import PectraImage from "@/public/images/roadmap/roadmap-pectra.png"
+
+interface Release {
+  image: StaticImageData
+  releaseName: string
+  releaseDate: string
+  content: React.ReactNode[]
+  href: string
+}
+
+export const releasesData: Release[] = [
+  {
+    image: DevelopersHubHeroImage,
+    releaseName: "Paris (The Merge)",
+    releaseDate: "2022-09-15",
+    content: [
```

Comment:
why is content an array? seems like we are always rendering one node

Reply:
Artifact of old idea for how I was going to do this, will refactor.